src/data/move-attrs/present-power-attr.ts

```diff
@@ -19,9 +22,9 @@ export class PresentPowerAttr extends VariablePowerAttr {
      * If this move is multi-hit, and this attribute is applied to any hit
      * other than the first, this move cannot result in a heal.
      */
-    const firstHit = user.turnData.hitCount === user.turnData.hitsLeft;
+    const isFirstHit = user.turnData.hitCount === user.turnData.hitsLeft;
 
-    const powerSeed = randSeedInt(firstHit ? 100 : 80);
+    const powerSeed = user.randSeedInt(isFirstHit ? 100 : 80);
     if (powerSeed <= 40) {
```

Comment:
```suggestion
    if (powerSeed < 40) {
```
...and so on for the other RNG thresholds